apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx

```diff
@@ -52,95 +72,47 @@ function AccountHeader({
     navigation.goBack();
   }, [navigation, readOnlyModeEnabled]);
 
-  const WalletTitleAnimation = useAnimatedStyle(() => {
-    const opacity =
-      currentPositionY.value === 0
-        ? 1
-        : interpolate(
-            currentPositionY.value,
-            [graphCardEndPosition - 40, graphCardEndPosition],
-            [1, 0],
-            Extrapolate.CLAMP,
-          );
-
-    return {
-      opacity,
-    };
-  }, [graphCardEndPosition]);
-
-  const PortfolioValueAnimation = useAnimatedStyle(() => {
-    const opacity = interpolate(
-      currentPositionY.value,
-      [graphCardEndPosition - 40, graphCardEndPosition - 30],
-      [0, 1],
-      Extrapolate.CLAMP,
-    );
-
-    return {
-      opacity,
-    };
-  }, [graphCardEndPosition]);
-
-  const BackgroundOpacity = useAnimatedStyle(() => {
-    const opacity = interpolate(
-      currentPositionY.value,
-      [graphCardEndPosition - 100, graphCardEndPosition - 90],
-      [0, 1],
-      Extrapolate.CLAMP,
-    );
-
-    return {
-      opacity,
-    };
-  }, [graphCardEndPosition]);
-
-  const Header = styled(Flex)`
-    position: absolute;
-  `;
-
-  const windowsWidth = getWindowDimensions().width;
   return (
-    <Header
-      flexDirection="row"
-      alignItems="center"
-      px={6}
-      py={4}
-      justifyContent="space-between"
-      width={windowsWidth}
-      height={92}
-      pt={44}
-    >
-      <Animated.View
-        style={[
-          {
-            position: "absolute",
-            width: windowsWidth,
-            height: 92,
-            overflow: "hidden",
-          },
-          BackgroundOpacity,
-        ]}
-      >
-        <Box height={"100%"} width={windowsWidth}>
-          <CurrencyGradient
-            gradientColor={getCurrencyColor(currency) || colors.primary.c80}
-          />
-        </Box>
-      </Animated.View>
-      <Box>
+    <CurrencyHeaderLayout
+      currentPositionY={currentPositionY}
+      graphCardEndPosition={graphCardEndPosition}
+      leftElement={
         <Touchable onPress={onBackButtonPress}>
           <ArrowLeftMedium size={24} />
         </Touchable>
-      </Box>
-      <Flex flexDirection={"row"} alignItems={"center"}>
-        <Text variant={"large"} fontWeight={"semiBold"}>
-          {account.name}
-        </Text>
-      </Flex>
-      <Box>
-        <AccountHeaderRight />
-      </Box>
-    </Header>
+      }
+      centerAfterScrollElement={
+        <Flex flexDirection={"column"} alignItems={"center"}>
+          {typeof items[1]?.value === "number" && account?.name ? (
+            <>
+              <Text
+                variant={"small"}
+                fontWeight={"semiBold"}
+                color={"neutral.c70"}
+                fontSize="11px"
+              >
+                {account.name}
```

Comment:
If we are on a token account we should display `parentAccount.name - account.name`